src/view/actions/sendEvent.jsx

```diff
@@ -179,15 +250,20 @@ const SendEvent = () => {
               </div>
             </div>
             <div className="u-gapTop">
-              <InfoTipLayout tip="Influences whether the SDK should retrieve and render personalization content, among other things.">
+              <InfoTipLayout tip="Influences whether the SDK should retrieve and render personalization content.">
```

Comment:
The scopes array influences which decisions should be retrieved, but this checkbox just determines if Alloy will render them right?

Reply:
this checkbox means `renderDecisions:true` which means retrieve decisions for page-wide scope and render visual experiences.